src/lib/journey/stages/step.mock.ts

```diff
@@ -1,5 +1,70 @@
+  callbacks: [
+    {
+      type: CallbackType.NameCallback,
+      output: [
+        {
+          name: 'prompt',
+          value: 'User Name',
+        },
+      ],
+      input: [
+        {
+          name: 'IDToken1',
+          value: '',
+        },
+      ],
+      _id: 69,
+    },
+    {
+      type: CallbackType.PasswordCallback,
+      output: [
+        {
+          name: 'prompt',
+          value: 'Password',
+        },
+      ],
+      input: [
+        {
+          name: 'IDToken2',
+          value: '',
+        },
+      ],
+      _id: 70,
+    },
+    {
+      type: CallbackType.DeviceProfileCallback,
+      output: [
+        {
+          name: 'metadata',
+          value: true,
+        },
+        {
+          name: 'location',
+          value: true,
+        },
+        {
+          name: 'message',
+          value: 'my message',
+        },
+      ],
+      input: [
+        {
+          name: 'IDToken3',
+          value:
+            '{"identifier":"2258096159-3136287393-51942348","metadata":{"hardware":{"cpuClass":null,"deviceMemory":8,"hardwareConcurrency":16,"maxTouchPoints":0,"oscpu":null,"display":{"width":3440,"height":1440,"pixelDepth":24,"angle":0}},"browser":{"userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36","appName":"Netscape","appCodeName":"Mozilla","appVersion":"5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36","appMinorVersion":null,"buildID":null,"product":"Gecko","productSub":"20030107","vendor":"Google Inc.","vendorSub":"","browserLanguage":null,"plugins":"internal-pdf-viewer;mhjfbmdgcfjbbpaeojofohoefgiehjai;internal-nacl-plugin;"},"platform":{"language":"en-US","platform":"MacIntel","userLanguage":null,"systemLanguage":null,"deviceName":"Mac (Browser)","fonts":"cursive;monospace;sans-serif;fantasy;Arial;Arial Black;Arial Narrow;Arial Rounded MT Bold;Comic Sans MS;Courier;Courier New;Georgia;Impact;Papyrus;Tahoma;Trebuchet MS;Verdana;","timezone":360}},"location":{"latitude":39.999445,"longitude":-105.0224336}}',
```

Comment:
Shouldn't this value property be `''`? This mock response should not have the profile pre-populated in it, like the username and password callback just above.